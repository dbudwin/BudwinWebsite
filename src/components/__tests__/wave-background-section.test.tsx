import '@testing-library/jest-dom'

import React from 'react'
import WaveBackgroundSection from '../wave-background-section'
import renderer from 'react-test-renderer'
import { faker } from '@faker-js/faker'

it('renders a wave background section', () => {
  const waveBackgroundSection = renderer.create(
    <WaveBackgroundSection id={faker.lorem.word()}>
      <React.Fragment></React.Fragment>
    </WaveBackgroundSection>
  ).toJSON()

  expect(waveBackgroundSection).toMatchSnapshot()
})
