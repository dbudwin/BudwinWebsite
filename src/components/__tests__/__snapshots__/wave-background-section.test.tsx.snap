// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a wave background section 1`] = `
<div>
  <section
    className="hero make-transparent is-fullheight-with-navbar"
  >
    <div
      className="hero-body"
    >
      <div
        className="container"
      />
    </div>
  </section>
</div>
`;
